### PR TITLE
[cmake] fix ios addon zip path after #16039

### PR DESCRIPTION
**Pull request by phunkyfish**
## Description

Fix for change in zip name from xbmc/xbmc#16039 causing ios addon builds to fail.

Thanks to @fuzzard for supplying the patch.

## How Has This Been Tested?

Currently have no way to test this. If someone could verify that would be great.

## Types of change

- [X] **Bug fix** (non-breaking change which fixes an issue)
- [ ] **Clean up** (non-breaking change which removes non-working, unmaintained functionality)
- [ ] **Improvement** (non-breaking change which improves existing functionality)
- [ ] **New feature** (non-breaking change which adds functionality)
- [ ] **Breaking change** (fix or feature that will cause existing functionality to change)
- [ ] **Cosmetic change** (non-breaking change that doesn't touch code)
- [ ] **None of the above** (please explain below)

## Checklist:

- [X] My code follows the **[Code Guidelines](https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md)** of this project 
- [ ] My change requires a change to the documentation, either Doxygen or wiki
- [ ] I have updated the documentation accordingly
- [X] I have read the **[Contributing](https://github.com/xbmc/xbmc/blob/master/docs/CONTRIBUTING.md)** document
- [ ] I have added tests to cover my change
- [ ] All new and existing tests passed

### CODE REVIEW

File: cmake/scripts/common/PrepareEnv.cmake
Comment (Rechi) [inline cmake/scripts/common/PrepareEnv.cmake]: I would have added `set(PLATFORM_TAG ${CORE_PLATFORM_NAME})` above this line and kept everything else as it was, to be more consistent with the other blocks of this if/elseif statement.

**Review by Rechi — APPROVED** [review]

File: cmake/scripts/common/PrepareEnv.cmake
Comment (phunkyfish) [reply cmake/scripts/common/PrepareEnv.cmake]: Cool, will change that.

**Comment by Rechi** [comment]
wrong revision was build
jenkins build this please

**Comment by phunkyfish** [comment]
If someone could merge this once Jenkins is happy please.

**Comment by phunkyfish** [comment]
Ha, and he’s happy

**Comment by Rechi** [comment]
@phunkyfish please always set milestone before or right after merging a PR.

**Comment by phunkyfish** [comment]
Of course, sorry, didn't realise ;)

**Pull request merged**
